### ISSUE TITLE
Simple mode that doesn't require constant internet?

### ISSUE THREAD

**Issue by lcalder** [matkatmusic/ChordieApp_IssueTracker#64]
I mentioned this in a different thread, but since it might be considered to be a new issue decided to reiterate in a separate issue tracker...

It would be nice to have a version of chordie that didn't require constant internet connection. First, the overhead of starting the program and waiting for the 5 to 10 seconds each time it needs to "call home..." gets frustrating, especially when I happens many times a day, and it just adds to an already complex process of loading all the software necessary to teach online...

I have read your reasons for the need for an online database and I agree the points made make sense from a jazz perspective where increasing complex chords need to be described in numerous ways, however it will expand the market for this software to teachers like me who teach theory and classical piano. I like to teach my students the pop/jazz labeling in addition to the Roman Numerals / Figured Bass as well as other labeling systems to describe various chords of the 7th, 9th, 11th and 13th (and the basic triads), some of which are too complicated for a program such as chordie because of a need to establish the "locally reigning tonic" throughout the music (Roman numerals), etc and this requires some human interpretation. Thus more "basic" easily-defined chords could be listed by chordie with needing the constant internet access to a constantly growing and correcting database which understandably works best in an online format.

As I mentioned in the other thread, there could be a basic mode or an auto mode whereby if lookup is possible it is displayed along with a green "light" somewhere in the window "status" area symbolizing an "working" internet connection as interpreted by chordie's needs (it often seems to complain there is no internet when there actually IS...) and then when the internet doesn't meet chordie's needs the status can display a red indicator on the "basic/simple" internally-generated chord label interpretation to show that it isn't a "complete" interpretation...

I would be willing to pay for a new license(s) or higher license fee to have such a feature enabled.

**Comment by T-Fizz** [comment]
I agree, needing a constant internet connection is a pain. I can't even use the app right now as their services seem to be down. What a waste.

**Comment by stavlocker** [comment]
Most certainly agreed, sometimes the service can be unresponsive which happens too often, and a user's internet connection may be spotty which both lead to the app being non-functional despite not really requiring the internet (especially when the app was already loaded and then stopped working mid-use).

Are there any plans to implement such a feature?
